scipy/special/_cephes.pxd

```diff
@@ -7,6 +7,7 @@ cdef extern from "cephes.h" nogil:
     double lbeta(double a, double b);
     double btdtr(double a, double b, double x);
     double cbrt(double x);
+    double chbevl(double x, double *array, int n);
```

Comment:
Do you have to use the array pointer instead of array[] in Cython header files? I'm just asking because this differs slightly from the other C function prototype declarations for `chbevl` elsewhere, although they are effectively the same thing (I think there can be a style preference to use *array or array[] to most clearly reflect the usage characteristics within the function itself).

Reply:
Both mean the same thing in prototypes in C.

Reply:
Yeah, I acknowledged that -- there are occasionally stylistic preferences, so just asking about the switch here.

Reply:
> just asking about the switch here

AFAIK, the `double array[]` syntax, while valid in C, is invalid in Cython. For example, the simple program

```
cdef double a[]
```

fails to compile with the complaint `Variable type 'double []' is incomplete`. Only fixed length array declarations are okay, i.e.

```
cdef double a[3]
```

This was the reason for the switch.

Reply:
But, it works fine in functions

```
cdef foo(double a[]):
    pass
```

so I'm mistaken. Can be changed if there is a strong desire.